### PR TITLE
feat: add darwin support

### PR DESCRIPTION
**Pull request by orthros**
Adds support for Darwin operating system.
Screen Shot 2021-10-05 at 12 21 32

### CODE REVIEW

File: cmd/info_darwin.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import (
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func OS() string {
+	out, err := exec.Command("sw_vers", "-productName").CombinedOutput()
+	if err != nil {
+		return "<Err getting OS Name>"
```
Comment (sammy-ette) [inline cmd/info_darwin.go]: should be `Unknown` instead

File: cmd/info_darwin.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	}
+	return strings.TrimSuffix(string(out), "\n")
+}
+
+func Kernel() string {
+	out, err := exec.Command("uname", "-r").CombinedOutput()
+	if err != nil {
+		return "<Err getting Kernel Version>"
```
Comment (sammy-ette) [inline cmd/info_darwin.go]: same thing

**Review by sammy-ette — CHANGES_REQUESTED** [review]

**Comment by sammy-ette** [comment]
otherwise nice one!

**Comment by orthros** [comment]
Thanks! I think I resolved your comments.

**Review by sammy-ette — APPROVED** [review]

**Pull request merged**